docs/rules/no-unused-expressions.md

```diff
@@ -59,6 +59,8 @@ delete a.b
 void a
 ```
 
+Strict mode directives are also not considered problems.
```

Comment:
Does our parser actually call out directives vs ExpressionStatements? If not, I think your suggested language will be even more confusing.